br/pkg/task/backup.go

```diff
@@ -113,6 +113,10 @@ func DefineBackupFlags(flags *pflag.FlagSet) {
 		"backup sst file compression algorithm, value can be one of 'lz4|zstd|snappy'")
 	flags.Int32(flagCompressionLevel, 0, "compression level used for sst file compression")
 
+	flags.Uint32(flagConcurrency, 4, "The size of thread pool on BR that executes tasks, "+
```

Comment:
```suggestion
	flags.Uint32(flagConcurrency, 4, "The size of a BR thread pool that executes tasks, "+
```
If `that executes tasks` follows `BR`, the subject is BR, not the thread pool.